app/controllers/staff/activity_forms_controller.rb

```diff
@@ -9,6 +9,8 @@ class Staff::ActivityFormsController < Staff::BaseController
     :sector,
     :status,
     :dates,
+    :geography,
```

Comment:
I think this name quite confusing. Perhaps calling it something like `geography_level` would make it clearer? Or even better would be to connect it to what it determines. I'd have (and did have) a hard time looking at this and following what `geography` represents.

Reply:
I really struggled with this. Is it the step name or the attribute name or both that were confusing? 

I settled on `geography` as that is the term IATI used to represent these concepts:

http://reference.iatistandard.org/203/activity-standard/overview/geography/

I went for what I felt was consistent with the rest of the model.

Reply:
I think it's both that are confusing. I get that IATI have a certain value, but I would hope to be able to read the code without having to refer to IATI docs to understand the meaning of fields.

Perhaps: `geographic_coverage` is a good name? That's wording used in the IATI docs. Or `geographic_scale`?